### DISCUSSION TITLE
skip-dirs not behaving as documented (trivy 0.30.0)

### DISCUSSION THREAD

**Discussion by mihob** [aquasecurity/trivy#2524] [Q&A] [answered]
The documentation says that multiple directories are ignored by repeating the `skip-dirs` parameter.

`trivy fs --skip-dirs /path/to/dir1 --skip-dirs /path/to/dir2 .`

But it looks like only the last `skip-dirs` is respected.

But the following syntax seems to work:

`trivy fs --skip-dirs "/path/to/dir1 /path/to/dir2" .`

Has the syntax changed in the current version? Or am I misunderstanding something here? I'm a little lost right now.

**Comment by afdesk** [comment, marked as answer]
@mihob thanks for your report!
there is a problem with this flag, I've created an issue, ok?

**Reply by mihob** [reply]
thank you!
